subsys/event_manager/Kconfig

```diff
@@ -41,16 +41,19 @@ config EVENT_MANAGER_LOG_EVENT_TYPE
 	bool "Include event type in the event log output"
 	default y
 
+config EVENT_MANAGER_MAX_EVENT_CNT
+	int "Maximum number of event types"
+	default 32
+	help
+	  Maximum number of declared event types in Event Manager.
+	  Note that there are two hidden events for tracking event processing start and stop in Event Manager.
```

Comment:
The relation is that when we track event manager events we need (in some configurations) two additional event.
So instead of adding any information in documentation I would create a build assert checking that there are enough profiler events to satisfy the event manager needs. In case something is wrong user will see error text telling what is not right.